.github/workflows/main.yml

```diff
@@ -222,7 +222,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8]
+        python-version: [3.8, '3.10']
```

Comment:
Maybe the 3.10 instead of 3.11 is due to dependents in the tutorials not being available? Cplex is now available #533 - so maybe once thats merged? Is there anything else  - that would have these done at the lowest and highest supported versions.

Reply:
Now that I merge #533, I will update the line to use 3.11 instead of 3.10.